forest/cli/src/cli/snapshot_cmd.rs

```diff
@@ -211,11 +215,40 @@ impl SnapshotCommands {
                     dry_run: *dry_run,
                 };
 
+                let bar = Arc::new(tokio::sync::Mutex::new({
+                    let bar = ProgressBar::new(0);
+                    bar.message("Exporting snapshot | blocks ");
+                    bar
+                }));
+                tokio::spawn({
+                    let bar = bar.clone();
+                    async move {
+                        let mut interval =
+                            tokio::time::interval(tokio::time::Duration::from_secs(1));
+                        loop {
+                            interval.tick().await;
+                            if let Ok((progress, total)) =
+                                get_progress((GetProgressType::SnapshotExport,), &None).await
+                            {
+                                let bar = bar.lock().await;
+                                if bar.is_finish() {
+                                    break;
+                                }
+                                bar.set_total(total);
+                                bar.set(progress);
+                            }
+                        }
+                    }
+                });
+
                 let out = chain_export(params, &config.client.rpc_token)
                     .await
                     .map_err(handle_rpc_err)?;
 
-                println!("Export completed. Snapshot located at {}", out.display());
+                bar.lock().await.finish_println(&format!(
```

Comment:
For some reason, I don't get a newline after this message. Do you see the same?

Reply:
This is what I get, could u attach ur console output?
```console
     Running `target/release/forest-cli snapshot export`
Exporting snapshot | blocks 1973583 / 1973583 [===================================================] 100.00 % 13369.41/s
Export completed. Snapshot located at ./forest_snapshot_calibnet_2023-05-09_height_531262.car./forest_snapshot_calibnet_2023-05-09_height_531262.car: OK
➜  forest git:(hm/walk-snapshot-progress-bar) ✗
```

Reply:
For me, it looks like this:
```
Exporting snapshot | blocks 2041991 / 2041991 [========================] 100.00 % 43995.89/s 
Export completed. Snapshot located at ./forest_snapshot_calibnet_2023-05-10_height_546622.carlemmih @ pop-os ~/coding/forest (hm/walk-snapshot-progress-bar)
└─ $ 
```

Reply:
It's strange, I've checked the call stack everything looks right, inside `pbr` it calls `writefln`

Reply:
Such mysteries make working with computers "fun". :)